### DISCUSSION TITLE
How do I access collection on external HD?

### DISCUSSION THREAD

**Discussion by MarlowMan93** [stashapp/stash#2851] [General]
Hi, only just got Stash and loving it so far! 

I store my collection on an external hard drive, how do I access the content via Stash? 

TYIA

**Comment by DogmaDragon** [comment]
Hi,

go to Stash, Settings > Library > Add Directory, and type in the drive letter. Then select the folder/folders you want. 
![image](https://user-images.githubusercontent.com/103123951/186409927-015b5cbd-fd89-46a8-b870-d0ce71fa6ea0.png)

**Reply by TaurnilCelebrindal** [reply]
Hello,

Thanks for sharing ;)

I also store my collection on many external hard drive (USB) , how do I access the content via Stash on Raspberry or linux mint ?

Each time I got an "error, files does not exist"
